frontend/app/routes/$lang/_public/apply/$id/child/exit-application.tsx

```diff
@@ -70,11 +70,11 @@ export default function ApplyFlowTaxFiling() {
       </div>
       <fetcher.Form method="post" noValidate className="flex flex-wrap items-center gap-3">
         <input type="hidden" name="_csrf" value={csrfToken} />
-        <ButtonLink id="back-button" routeId="$lang/_public/apply/$id/child/review-adult-information" params={params} disabled={isSubmitting} data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form:Back - Exiting the application click">
+        <ButtonLink id="back-button" routeId="$lang/_public/apply/$id/child/review-adult-information" params={params} disabled={isSubmitting} data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form-Child:Back - Exit application click">
```

Comment:
The page title is `Exiting the application`.

Reply:
Let's continue with you proposed change. `Exiting application`. 👍

Reply:
Updated it in the commit, let me know if you want me to revert it